### PR TITLE
[node] bingo with signature information

### PR DESCRIPTION
**Pull request by gupadhyaya**
fixes https://github.com/harmony-one/harmony/issues/2771

### CODE REVIEW

**Comment by gupadhyaya** [comment]
local output:
```
./zerolog-validator-127.0.0.1-9003.log:1286:{"level":"info","port":"9003","ip":"127.0.0.1","blockNum":22,"epochNum":3,"ViewId":22,"blockHash":"0x3d6f84925bed36b9449d54bc04799bc6cb79e11d6249ad0cca33adbfe1551449","numTxns":0,"numStakingTxns":0,"caller":"/Users/gupadhyaya/go/src/github.com/harmony-one/harmony/node/node_handler.go:480","time":"2020-04-09T11:24:21.283968-07:00","message":"BINGO !!! Reached Consensus, signatures included: ee2474f93cba9241562efc7475ac2721ab0899edf8f7f115a656c0c1f9ef8203add678064878d174bb478fa2e6630502"}
```

File: node/node_handler.go [hunk 1]
```diff
@@ -443,14 +464,20 @@ func (node *Node) PostConsensusProcessing(
...
+			}
 			utils.Logger().Info().
 				Uint64("blockNum", newBlock.NumberU64()).
 				Uint64("epochNum", newBlock.Epoch().Uint64()).
 				Uint64("ViewId", newBlock.Header().ViewID().Uint64()).
 				Str("blockHash", newBlock.Hash().String()).
 				Int("numTxns", len(newBlock.Transactions())).
 				Int("numStakingTxns", len(newBlock.StakingTransactions())).
-				Msg("BINGO !!! Reached Consensus")
+				Msgf("BINGO !!! Reached Consensus, signatures included: %s", sigs)
```
Comment (rlan35) [inline node/node_handler.go:480]: Better to only show "BINGO !!! Reached Consensus; Signatures Included: %d", len(sigs)

No one really care about the signature detail. Just whether it's included or not.

File: node/node_handler.go [hunk 2]
```diff
@@ -443,14 +464,20 @@ func (node *Node) PostConsensusProcessing(
 		node.BroadcastCXReceipts(newBlock, commitSigAndBitmap)
 	} else {
 		if node.Consensus.Mode() != consensus.Listening {
+			sigs := "none"
+			if sigsInBlock, err := node.signaturesIncludedInBlock(newBlock); err == nil {
```
Comment (fxfactorial) [inline node/node_handler.go:468]: While this PR open and you have this code, can you do me a favor and compare the time of compute needed to call this block.

Comment (gupadhyaya) [reply node/node_handler.go:468]: between 12-60 microseconds in local settings with one key per node.

Comment (gupadhyaya) [reply node/node_handler.go:468]: 5-25 microseconds after removing the SerializeToHexStr.

Comment (fxfactorial) [reply node/node_handler.go:468]: sorry - since so easy now - can also say when under multi bls key, if that causes any extra slowdown

Comment (fxfactorial) [reply node/node_handler.go:468]: actually nvm - at the level of microsecond - doesn't matter , all good

**Pull request merged**